src/ServiceProviders/CIProviders/BitbucketPipelines/BitbucketPipelinesProvider.php

```diff
@@ -109,10 +109,32 @@ public function startTesting(CIState $ci_env)
         // Temporary: catch and eat errors without stopping the command
         try
         {
+            // Enable Pipelines for the project.
             $this->api()->request("$repoApiUrl/pipelines_config", $data, 'PUT');
         } catch (\Exception $e) {
             $this->logger->error($e->getMessage());
         }
+
+        // @TODO: verify that the custom "clu" task exists in yml first?
+        $data = [
+          'enabled' => true,
+          'cron_pattern' => '0 0 4 * * ? *',
```

Comment:
This cron pattern seems odd. Is this what worked for you?

Reply:
I was expecting a more regular pattern like `0 0 * * *` for "every day at midnight"

Reply:
Yeah, it's crazy. I reverse engineered this statement by calling the GET endpoint to see what a manually created job looks like.

Based on that, I think maybe it follows Jira's synatx: https://confluence.atlassian.com/jiracorecloud/constructing-cron-expressions-for-a-filter-subscription-765593786.html

Reply:
Okay, so following that example `0 0 4 * * ? *` is every day at 4 am. The `?` is needed for _day of the week_ to "Specify no particular value."

Reply:
Maybe this is a good opportunity to open a PR to the BitBucket Pipelines API docs for Hacktoberfest